src/benchmarks/transformbench/transformbench.cc

```diff
@@ -0,0 +1,602 @@
+
+#include "mra/misc/platform.h"
+#include "mra/kernels/transform.h"
+#include "mra/tensor/tensor.h"
+#include "mra/tensor/tensorview.h"
+#include "mra/misc/options.h"
+#include "mra/misc/types.h"
```

Comment:
Vanity comment:
``` suggestion
#include "mra/misc/types.h"
#include "mra/misc/options.h"
#include "mra/tensor/tensor.h"
#include "mra/misc/platform.h"
#include "mra/kernels/transform.h"
#include "mra/tensor/tensorview.h"
```

Reply:
Why?

Reply:
Just being consistent with ordering for vanity. Rather
``` suggestion
#include "mra/misc/types.h"
#include "mra/misc/options.h"
#include "mra/misc/platform.h"
#include "mra/tensor/tensor.h"
#include "mra/kernels/transform.h"
#include "mra/tensor/tensorview.h"
```